test/xbgpu/test_bsend.py

```diff
@@ -35,9 +35,9 @@
 
 from . import test_parameters
 
-FRAMES_PER_CHUNK: Final[int] = 5
-N_TX_ITEMS: Final[int] = 2
-TX_HEAPS_PER_SUBSTREAM: Final[int] = N_TX_ITEMS * FRAMES_PER_CHUNK
+BATCHES_PER_CHUNK: Final[int] = 5
+N_OUT_ITEMS: Final[int] = 2
```

Comment:
I'd suggest calling this N_CHUNKS, since it's controlling allocation of Chunks not BOutQueueItems.